### PR TITLE
Normalize references to CLI commands in documentation.

### PR DESCRIPTION
**Pull request by glean-calder**
...and any other typos I found along the way.

### CODE REVIEW

File: pages/docs/data-ops/cli/cache.mdx [hunk 1]
```diff
@@ -1,16 +1,15 @@
 import { Callout, Steps } from "nextra-theme-docs";
 
-# Hashboard cache command
+# `cache`
```
Comment (trucklos) [inline pages/docs/data-ops/cli/cache.mdx:3]: @glean-calder  heads up I actually think having "Hashboard cache command" as the heading is better for SEO

Comment (glean-calder) [reply pages/docs/data-ops/cli/cache.mdx:3]: Hmm...why exactly / do we know how much better?

Taking this to the fullest extreme, we could put `hashboard` in every title on our docs page 🤷

**Comment by glean-calder** [comment]
Bumping this!

File: pages/docs/data-ops/cli/cache.mdx (hunk 1 above)
Comment (trucklos) [reply pages/docs/data-ops/cli/cache.mdx:3]: I don't know exactly how much better.  If someone just lands on our caching page (through SEO or some other deep link), it is a bit confusing to just have the title be 'cache' and without much additional context.  Example of how other popular tools document their CLI's: https://docs.getdbt.com/reference/commands/compile

Why do we have to take it to an extreme, can we just keep these titles please?

**Review by trucklos — CHANGES_REQUESTED** [review]
See my comment below.  https://github.com/hashboard-hq/documentation/pull/51/files#r1326002545

File: pages/docs/data-ops/cli/cache.mdx (hunk 1 above)
Comment (glean-calder) [reply pages/docs/data-ops/cli/cache.mdx:3]: Good enough for me! Will change the titles back

**Review by trucklos — APPROVED** [review]
👏  thank you!

**Pull request merged**